stdlib/public/core/FloatingPointTypes.swift.gyb

```diff
@@ -760,25 +761,33 @@ extension ${Self} {
 extension ${Self} {
 % for srcBits in allFloatBits:
 %   That = floatName[srcBits]
-%   if Self != That:
 
-%     if srcBits == 80:
+%   if srcBits == 80:
 #if !os(Windows) && (arch(i386) || arch(x86_64))
-%     end
+%   end
 
   /// Construct an instance that approximates `other`.
```

Comment:
Please restore the old comment for the case when `Self` is `That` (`Create an instance initialized to ...`).